versioned_docs/version-8.7/self-managed/modeler/web-modeler/configuration/configuration.md

```diff
@@ -26,16 +26,29 @@ The Camunda 8 [Helm](/self-managed/setup/install.md) and [Docker Compose](/self-
+| `CAMUNDA_MODELER_CLUSTERS_0_VERSION`                 | The Camunda version used by this cluster.                                                                      | `8.7.0`                                                                     |
+| `CAMUNDA_MODELER_CLUSTERS_0_AUTHENTICATION`          | The [authentication](#available-authentication-methods) to use with your cluster.                              | `BEARER_TOKEN`                                                              |
+| `CAMUNDA_MODELER_CLUSTERS_0_URL_ZEEBE_GRPC`          | [Internal or external](#notes-on-host-names-and-port-numbers) address where your cluster can be reached.       | `grpc://camunda-zeebe-gateway:26500`, `grpcs://camunda-zeebe-gateway:26500` |
+| `CAMUNDA_MODELER_CLUSTERS_0_URL_ZEEBE_REST`          | [Internal or external](#notes-on-host-names-and-port-numbers) address where the Zeebe REST API can be reached. | `http://camunda-zeebe-gateway:8080`, `https://camunda-zeebe-gateway:8080`   |
+| `CAMUNDA_MODELER_CLUSTERS_0_URL_OPERATE`             | [Internal or external](#notes-on-host-names-and-port-numbers) address where Operate can be reached.            | `http://camunda-operate:80`, `https://camunda-operate:80`                   |
+| `CAMUNDA_MODELER_CLUSTERS_0_URL_TASKLIST`            | [Internal or external](#notes-on-host-names-and-port-numbers) address where Tasklist can be reached.           | `http://camunda-tasklist:80`, `https://camunda-tasklist:80`                 |
+| `CAMUNDA_MODELER_CLUSTERS_0_OAUTH_URL`               | The address of your token issuer. Required for authentication `CLIENT_CREDENTIALS`.                            | `https://auth.example.com/token`                                            |
+| `CAMUNDA_MODELER_CLUSTERS_0_OAUTH_SCOPE`             | A comma-separated list of the scopes to use with this cluster.                                                 | `test-scope`                                                                |
+| `CAMUNDA_MODELER_CLUSTERS_0_OAUTH_AUDIENCE_ZEEBE`    | The permission name for Zeebe.                                                                                 | `zeebe-api`                                                                 |
+| `CAMUNDA_MODELER_CLUSTERS_0_OAUTH_AUDIENCE_OPERATE`  | The permission name for Operate.                                                                               | `operate-api`                                                               |
+| `CAMUNDA_MODELER_CLUSTERS_0_OAUTH_AUDIENCE_TASKLIST` | The permission name for Tasklist.                                                                              | `tasklist-api`                                                              |
+
+#### Available authentication methods
```

Comment:
💯 Very useful addition!

Why didn't you add this to the 8.8 configuration, too?

Reply:
For 8.8, I was a bit unsure about the wording given that the criteria "identity provider supports access tokens with multiple audiences" will no longer be relevant. Also, we probably need to reflect the changes from the architecture streamlining initiative and the changes in the security configuration (see https://docs.camunda.io/docs/next/self-managed/identity/orchestration-identity/configuration/). There's now `CAMUNDA_SECURITY_AUTHENTICATION_METHOD` (which can be `oidc` or `basic`) and a separate property `CAMUNDA_SECURITY_AUTHENTICATION_UNPROTECTEDAPI` to allow unauthenticated access to the API.

I would suggest to come back to this topic and add the table for 8.8, when we add support for basic authentication to Web Modeler.

Reply:
Generally, Web Modeler is backward-compatible, see https://docs.camunda.io/docs/next/reference/supported-environments/#component-requirements

> You can also use newer versions of Desktop and Web Modeler with older Zeebe versions.

which is why I'd still opt for making the change now in 8.8 docs and adapting to the 8.8 reality when we tackle https://github.com/camunda/web-modeler/issues/12309

Reply:
Ok, true. Then I will just add the same table as in 8.7.